packages/client/src/app/pages/Login/Login.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FormEventHandler, MouseEventHandler } from 'react'
+import styles from './Login.module.scss'
+import Button from '../../components/core/Button/Button'
+import TextField from '../../components/core/TextField/TextField'
+import PasswordField from '../../components/core/PasswordField/PasswordField'
+import { Link } from 'react-router-dom'
+
+const Login = () => {
+
+  const submitHandler:FormEventHandler<HTMLFormElement> = (e): void => {
```

Comment:
```suggestion
  const submitHandler:FormEventHandler<HTMLFormElement> = (event): void => {
```
Давай будем делать код читаемым, а не угадываемым)

Reply:
Вероятно так действительно лучше

Reply:
Я не работал в других средах, но в Webstorm это выглядит так. И тут гадать не надо
![image](https://user-images.githubusercontent.com/50448502/205485431-d323a2e0-bba7-4313-9b70-11dd3c6bb5a0.png)